app/src/main/java/io/github/droidkaigi/confsched2017/MainApplication.java

```diff
@@ -39,6 +40,9 @@
     NotificationStrategy notificationStrategy;
 
     @Inject
+    ShowSplashStrategy showSplashStrategy;
```

Comment:
I think it's not good to add many debug menu to debot 🤔 
No need to fix on this PR. But I might arrange the debug / developer menu later.